src/LanguageServer/Impl/Implementation/Server.cs

```diff
@@ -249,6 +252,12 @@ public sealed partial class Server : IDisposable {
         }
 
         private void ResetAnalyzer() {
+            if (!_initialized) {
```

Comment:
another approach could be making this to be async method and await until initialized to happen and then proceed. but I saw caller of this thing and how things connected so might be too much work for a corner case.

basically, it looks like this can happen if files in path server watching (such as lib directory and etc) changed before initialization finished. and it those case, ignoring those changes seems fine to me.

Reply:
Yeah, we don't have to go overkill because if it's still being initialized, then any future reads of library files will necessarily be _after_ them being changed, so we'll get the updated info anyway. Reloading is more of a "everything you have before now is wrong, reset", but before init this is meaningless.